scripts/scene_elements/item.py

```diff
@@ -0,0 +1,49 @@
+import dataclasses
+from typing import List, Dict
+
+import snecs
+
+from scripts.core.data import Data
+from scripts.core.effect import load_effect
+
+
+@dataclasses.dataclass
+class ItemData:
+    """
+    Schema for the YAML Item data files
+
+    """
+    name: str
+    is_signature: bool
+    effects: List[Dict[str, str]]
+
+
+class Item(snecs.RegisteredComponent):
+    """
```

Comment:
I don't think that `Item` can be a component as components must be unique and this approach would suggest that it is 1 component per item.